frontend/src/app/modules/website/website.component.ts

```diff
@@ -0,0 +1,112 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { CreateEntityModalData } from '../shared/components/entity-create-modal/entity-create.types';
+import { Entity, Status } from '../shared/components/entity-list/entity-list.component';
+import { EntityService } from '../shared/services/entity.service';
+import { WebsiteService } from './website.service';
+
+@Component({
+  selector: 'app-website',
+  templateUrl: './website.component.html',
+  styleUrls: ['./website.component.scss']
+})
+export class WebsiteComponent implements OnInit {
+
+  baseEditUrl = '/dashboard/website/edit';
+
+  websites: Entity[] = [];
+
+  entityTitle = 'Websites';
+
+  createModal: Observable<{title: string, description: string}> = {} as Observable<{title: string, description: string}>;
+
+  constructor(
+    private websiteService: WebsiteService,
+    private entityService: EntityService
+  ) {}
+
+  ngOnInit(): void {
+    this.loadWebsitesList();
+    this.listenToCreateWebsite();
```

Comment:
these listeners would be part of entity list component and deal with entity list service